packages/nx/src/command-line/migrate.spec.ts

```diff
@@ -300,6 +300,7 @@ describe('Migration', () => {
         packageUpdates: {
           parent: { version: '2.0.0', addToPackageJson: false },
           child: { version: '2.0.0', addToPackageJson: false },
+          grandchild: { version: '2.0.0', addToPackageJson: false },
```

Comment:
What does adding this do? No assertions changed

Reply:
It's basically a no-op because grandchild is already at 2.0.0.. but it gets returned as a packageUpdate after these changes..

The test could probably be updated in a better way :thinking: